src/kata2.booleanToWord.js

```diff
@@ -1,5 +1,10 @@
 const booleanToWord = (boolean) => {
-
-}
+    
+    if (boolean) {
+        return 'Yes';
+    } else {
+        return 'No';
+    }
```

Comment:
You could do this with a [ternary](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) (a shorthand if / else statement):

```js
return boolean ? 'Yes' : 'No'
```